### ISSUE TITLE
Filter Curve EQ - User Presets not set when selected

### ISSUE THREAD

**Issue by belisoful** [audacity/audacity#7651]
### Steps to reproduce

1. Make a test tone or noise
2. Select the Menu "Effect" > "EQ and Filters" > "Filter Curve EQ"
3. Make a custom EQ curve
4. Click "Presets & settings" > "Save Preset..."
5. Enter a New User Present name.
6. Click "Presets & settings" > "Factory Presets" > "RIAA" to change the curve.
7. Click "Presets & settings" > "User Presets" and select your preset from No. 5
8. **BUG**. The user preset is not set.

### Expected behavior

in step 8, I was expecting the user preset to fill the visual graph and replace the RIAA curve with the User Preset.

In this way, All user presets do not work for Filter Curve EQ.
The Factor Presets do work.

### Actual behavior

When selecting a User Preset for Filter Curve EQ, nothing happens.  The preset is not filled into the filter.

### Audacity Version

Audacity 3.7.x

### Operating system

Windows, macOS

### Additional context

This bug is both in the MacOS version and Windows version.  The user presets for Filter Curve EQ have imploded.

**Comment by dozzzzer** [comment]
I couldn't reproduce this on Windows and MacOS in 3.7.0:

https://github.com/user-attachments/assets/03b6a183-4f3b-4f87-bc24-d7beb6626508

**Comment by belisoful** [comment]
I figured it out.   The preset name must have a "/".... then it never works.

 I was naming my presets like this: "-3db / octave" and "-6db / octave".   the "/ octave" is not showing up and is what is killing the preset.

If I don't add the "/" in the preset name, it works fine.

**Comment by dozzzzer** [comment]
I agree that the slash should be converted or removed. Let me update the ticket then.

**Comment by belisoful** [comment]
The "/" preset issue may be a "feature" across all plugins with presets.   I haven't tested any of the other audio effects' user presets for this bug, but I suspect they have it too.

**Comment by belisoful** [comment]
Don't forget any other characters that the file system may not like,  such as "~" or ":".

I haven't tested these other characters, but if the string is going to be scrubbed or converted (or mapped) for "/", then the other characters should be checked and if/when other characters are an issue, they should also be processed like "/"
